frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/TaskStatus.java

```diff
@@ -41,7 +41,7 @@
   private String containerHost;
   // containerIp is the assigned ipv4 address of the corresponding containerHost
   private String containerIp;
-  private List<ValueRange> containerPorts;
+  private String containerPorts;
   // containerGpus is the assigned GpuAttribute of the container
```

Comment:
Add comments:
// containerPorts is a string contains ports for each port label, such as "httpPort:80,81,82;sshPort:1021,1022,1023;"